Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectGenerator/Scripts/Exporters/IUnityProjectExporter.cs

```diff
@@ -46,11 +46,20 @@ public interface IUnityProjectExporter
         ICommonPropsExporter CreateCommonPropsExporter(FileInfo path);
 
         /// <summary>
-        /// Exports the Common props file based on the given compilation platform and whether to export it as an In-Editor flavor vs Player.
+        /// Creates the platform props file exporter.
         /// </summary>
-        /// <param name="platform">The platform to export.</param>
-        /// <param name="inEditorConfiguration">True if this is an In-Editor flavor, false otherwise.</param>
-        void ExportPlatformPropsFile(CompilationPlatformInfo platform, bool inEditorConfiguration);
+        /// <param name="path">The <see cref="FileInfo"/> representing where this props file will be written.</param>
+        /// <param name="unityConfiguration">The configuration for the platform props.</param>
+        /// <param name="unityPlatform">The unity platform for the platform props.</param>
+        /// <param name="scriptingBackend">The scripting backend for the platform props.</param>
+        IPlatformPropsExporter CreatePlatformPropsExporter(FileInfo path, string unityConfiguration, string unityPlatform, ScriptingBackend scriptingBackend);
+
+        /// <summary>
+        /// Creates the specialized platform props file exporter for Player|WSA combination.
+        /// </summary>
+        /// <param name="path">The <see cref="FileInfo"/> representing where this props file will be written.</param>
+        /// <param name="scriptingBackend">The scripting backend for the platform props.</param>
+        IWSAPlayerPlatformPropsExporter CreateWSAPlayerPlatformPropsExporter(FileInfo path, ScriptingBackend scriptingBackend);
```

Comment:
>CreateWSAPlayerPlatformPropsExporter [](start = 40, length = 36)

are we going to need a custom exporter for each platform/player combination? Are there specific things needed on IWSAPlayerPlatformPropsExporter that wouldn't pertain to other platforms?

Reply:
Yes, and yes Min/Max target UWP platform.